roles/ci_restore_snapshot/vars/Seapath_host_Yocto_distribution.yml

```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2024, RTE
+# SPDX-License-Identifier: Apache-2.0
+
+---
+grub_update_command: "update-grub"
```

Comment:
This is not useful, we will not use the snapshot restore in Yocto CI.

Otherwise, LGTM